src/Symfony/Component/FeatureToggle/Strategy/OuterStrategiesInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\FeatureToggle\Strategy;
+
+interface OuterStrategiesInterface extends StrategyInterface
```

Comment:
What is the goal of those interfaces exposing inner implementation details ?

Reply:
It was based on this https://www.php.net/manual/en/iteratoriterator.getinneriterator.php . Makes it both clear that it "composes" other strategies and easier to debug / trace. How would you suggest to change it ?

Reply:
Indeed. I thought we were using it already on Traceable but seems not. If you look at the screenshots (where we show the available features and their strategies) I'd like to improve it a bit further and thought this could come in handy. Will remove for the moment then.

Reply:
removed